modules/caddyhttp/fileserver/browse.go

```diff
@@ -146,6 +150,89 @@ func (fsrv *FileServer) browseWriteHTML(listing browseListing) (*bytes.Buffer, e
 	return buf, err
 }
 
+func (fsrv *FileServer) streamFolderAsZip(baseFolderName, downloadFolderName string, w http.ResponseWriter) error {
+	w.Header().Set("Content-type", "application/zip")
```

Comment:
Of course, I know the first question we're going to get is requests for other formats: tar.gz, tar.bz2, and .tar.zstd among the more likely I think.